include/experimental/__p1673_bits/conjugated.hpp

```diff
@@ -94,10 +94,21 @@ class conjugated_accessor {
 template<class ElementType, class Extents, class Layout, class Accessor>
 auto conjugated(mdspan<ElementType, Extents, Layout, Accessor> a)
 {
-  if constexpr (std::is_arithmetic_v<std::remove_cv_t<ElementType>>) {
+  using value_type = typename decltype(a)::value_type;
+
+  if constexpr (std::is_arithmetic_v<value_type>) {
```

Comment:
#ifdef should be somehwhere here, p1673 spec has nothing about arithmetic types. Only else branch is valid in 1673 context

Reply:
Good point!  I'll fix this.

Reply:
I just pushed a fix.  The tests now cover this case.